polar-core/src/vm.rs

```diff
@@ -1976,6 +1941,29 @@ impl PolarVirtualMachine {
             right_value = self.value(right_sym).cloned();
         }
 
+        // Rebind the previous name of `term` to a variable pointing to
+        // `var`. `term` must be a partial.
+        //
+        // Return: A new partial with the name updated to `var`.
+        let mut rebind_partial = |var: &Symbol, term: &Term| {
```

Comment:
Is there a reason this has to be a closure vs. defining a method on `Partial`?

Reply:
Yes, it calls methods on the vm.